src/applications/burials-ez/BurialsApp.jsx

```diff
@@ -55,8 +55,16 @@ export default function BurialsApp({ location, children }) {
     return <NoFormPage />;
   }
 
+  // Temporary overwrite of version until flipper is removed.
+  const ldFormConfig = !burialLocationOfDeathUpdate
+    ? formConfig
+    : {
+        ...formConfig,
+        version: 3,
```

Comment:
This forces the version change to occur (if the flipper is enabled) _before_ the saved form is requested, which triggers the migration.